main.js

```diff
@@ -168,3 +168,30 @@ detailCardClose4.addEventListener('click', () => {
 });
 // Card 4 popup opening closing ends Here
 // Adding Dynamic Data to Works ul ends Here
+
+// Contact Form Validation Starts Here
+// This Function will show error message
+const contactForm = document.getElementById('contact-me-form');
+const contactFormError = document.getElementById('contact-me-form-error-msg');
+const emailRegex = /^[_a-z0-9-]+(\.[_a-z0-9-]+)*@[a-z0-9-]+(\.[a-z0-9-]+)*(\.[a-z]{2,4})$/;
+
+contactForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+
+  if (contactForm.elements.name.value.trim() === '') {
+    contactFormError.innerText = 'Please enter your name';
+    contactFormError.classList.add('contact-me-form-error-msg-display');
+  } else if (!emailRegex.test(contactForm.elements.email.value.trim())) {
+    contactFormError.innerText = 'Please enter your email in lower case';
+    contactFormError.classList.add('contact-me-form-error-msg-display');
+  } else if (contactForm.elements.message.value.trim() === '') {
+    contactFormError.innerText = 'Please enter your message';
+    contactFormError.classList.add('contact-me-form-error-msg-display');
+  } else {
+    contactFormError.innerText = '';
+    contactFormError.className = 'contact-me-form-error-msg-hidden';
+    contactForm.submit();
+  }
```

Comment:
- [ ] For the project requirements, once you have fixed the above issue, make sure the error message is showing up close to the `submit` button or the `input fields`. (i.e error message can show up around the input fields or just around the submit button)